packages/design-system-react-native/src/components/Checkbox/Checkbox.tsx

```diff
@@ -20,7 +18,7 @@ const AnimatedView = Animated.View;
 export const Checkbox = forwardRef<{ toggle: () => void }, CheckboxProps>(
   (
     {
-      isSelected = false,
+      isSelected,
```

Comment:
Since `isSelected` is now a required prop (no longer has a default value), this enforces that Checkbox is always used as a controlled component. This change ensures parent components must explicitly manage its state, which improves clarity and avoids any ambiguity between controlled and uncontrolled usage.